internal/dataplane/parser/translate_ingress_test.go

```diff
@@ -425,7 +433,8 @@ func TestFromIngressV1beta1(t *testing.T) {
 			},
 		})
 		require.NoError(t, err)
-		p := NewParser(logrus.New(), store)
+		p, err := NewParser(logrus.New(), store)
```

Comment:
Can we make a `mustCreateParser` here to reduce the duplicate `require.NoError`?

Reply:
65b7862